packages/x-translations/.versionrc.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
```

Comment:
Do we actually need this file? Been checking the other packages an none of them have a `.versionrc.js` file.

- [ ] Check if we need this file.